scripts/tasks/perf-test.js

```diff
@@ -104,15 +104,26 @@ async function runScenarioNTimes(page, times) {
 
 function createBlobFromResults(perfBlob) {
   return `Component Perf Results:
-  | Scenario Name | Current Avg Total | New Avg Total | Current Avg Per Item | New Avg Per Item |
-  |----------|-------------------|---------------|----------------------|------------------|\n`.concat(
+  <table>
+  <tr>
+    <th>Scenario</th>
+    <th>Current Avg Total</th>
+    <th>New Avg Total</th>
+    <th>Current Avg Per Item</th>
+    <th>New Avg Per Item</th>
+  </tr>`.concat(
     Object.keys(perfBlob.now)
       .map(
         scenario =>
-          `| ${scenario}ms | ${perfBlob.now[scenario].total}ms | ${perfBlob.new[scenario].total}ms| ${perfBlob.new[scenario].peritem}ms | ${
-            perfBlob.new[scenario].peritem
-          }ms|`
+          `<tr>
+            <td>${scenario}</td>
+            <td>${perfBlob.now[scenario].total} ms</td>
+            <td>${perfBlob.new[scenario].total} ms</td>
+            <td>${perfBlob.now[scenario].peritem} ms</td>
+            <td>${perfBlob.new[scenario].peritem} ms</td>
```

Comment:
Minor comment, I would consider putting the unit of measure "ms" in the table headers, then you don't have to render them per cell per row. This is awesome tho!!

Reply:
I did consider that! done :)